packages/sdk/src/contracts/FuseUniswapContract.ts

```diff
@@ -107,16 +106,16 @@ export async function swap(
     onSent?: (transactionHash: string, from: string) => void,
     deadline: number = DEFAULT_DEADLINE_FROM_NOW // in seconds from now
 ): Promise<any> {
-    const chainId = await getChainId(web3)
     const account = await getAccount(web3)
 
     const contract = await fuseUniswapContract(web3)
     const parameters = await swapCallArguments(web3, trade, allowedSlippage, deadline)
 
     const { methodName, args, value } = parameters
+    const gasPrice = await web3.eth.getGasPrice()
 
     const req = contract.methods[methodName](...args).send({
-        ...(chainId === SupportedChainId.FUSE && { gasPrice: 1_000_000_000 }),
+        ...{ gasPrice: gasPrice },
```

Comment:
I believe it better to not rely on default, and go with the median so we gradually grow with other dapps pushing the 10 gwei.